### PR TITLE
Website completed

### PR DESCRIPTION
**Pull request by Ndaboom**
**Summit website - Home & About page completed** 
- Linters setup
- Home and about page fully designed (Mobile and desktop friendly)
- Project explanation videos at this [link](https://www.loom.com/share/6e28315501324ffcbdd9f9f9f1a511f1)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        </div>
+    </section>
+    <section class="footer_section">
+        <div class="summit_logo">
+            <img src="assets/images/summit-logo.png" alt="Summit logo" />
+        </div>
+        <div class="summit_adress">
+            <p>2015 Creative Commons Korea.<br>Some Rights Reserved.</p>
+        </div>
+    </section>
```
Comment (1GORDON) [inline about.html:105]: Great design @Ndaboom 🤝

- [x] Kindly make your design a close replica of the design provided. You may make the footer of the about section dark for the desktop version, and the footer for the mobile version white.
- Desktop Version
design-footer
- Mobile version
mobile-footer

File: about.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <script src="https://kit.fontawesome.com/b39c945343.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <section class="about_main_section">
+        <!-- Mobile header -->
+        <header id="mobile_header">
+            <nav class="toggleNav">
+                <img src="assets/icons/union.png" alt="union icon" />
```
Comment (1GORDON) [inline about.html]: Great design @Ndaboom 🤝

- [x] Kindly add `alt` attributes to all images across all your HTML files. this is a good practice that can help with screen readers.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Ndaboom ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] As stated in the project requirements, kindly showcase some interesting piece of code or something you built that you are very proud of in the video presentation.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @Ndaboom, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] Please know that you should use `h1` tag for the main title on each page, I can see that your main title is an image instead, I recommend that you turn it to text and put it in an `h1` tag.
- [ ] Please know that there is a white margin on top of the bar, I recommend you look into it and fix it:
image

- [ ] Please know that the logo in the footer isn't visible enough now that the background is dark, I suggest you fix that:
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**